salt/tests/unit/modules/files/test_metalk8s_cordon.yaml

```diff
@@ -0,0 +1,677 @@
+
+CORDON_OBJECT_RESULT:
```

Comment:
This file is really too huge for no good reason please just keep some usefull field and that it (like unschedulable and taints + some metadata like name)

Reply:
Done